docs/src/increase-observability/metrics.md

```diff
@@ -0,0 +1,207 @@
+
+## Examples of how to read metrics
+
+These examples should give you an idea of how the metrics appear by environment type.
+While architectural differences mean the data is represented differently depending on the environment type,
+you always find metrics on usage of CPU, RAM, and disk space.
+
+### Dedicated environment example
+
+Dedicated environments metrics show the resource usage per host within the cluster.
+
+This reference project has configured a single Python application and three services: Elasticsearch, Redis, and MariaDB.
+It has been granted 4&nbsp;GB of memory and 2 vCPUs per host based on its plan.
+
+![All of the metrics available for a Dedicated environment](/images/metrics/all.png "0.5")
+
+#### CPU
+
+The project has 2 virtual CPUs (vCPUs) allocated per host, as seen in the upper limit for the CPU metric.
+
+![A closeup of the CPU metrics](/images/metrics/cpu.png "0.5")
+
+#### RAM
+
+RAM measures overall RAM usage, displaying the percentage of the RAM available for each host.
+
+For this example, each host has been granted 4&nbsp;GB of memory.
+The upper limit in the graph is 3.62&nbsp;GB because 0.38&nbsp;GB of memory are in use by the operating system.
+
+![A closeup of the RAM metrics](/images/metrics/ram.png "0.5")
+
+#### Disk
+
+Disk represents the plan space for all services and mounts,
+which for Dedicated production environments is [50&nbsp;GB](../dedicated/architecture/_index.md).
+
+This example has the following persistent disk space configured:
+
+* App: 1024&nbsp;MB for the mount `exports`
+* Elasticsearch: 1024&nbsp;MB
+* MySQL: 2048&nbsp;MB
+* Redis: no persistent disk
+
+![All of the metrics for disks](/images/metrics/disk.png)
+
+For each host, you first see metrics for the root (`/`) folder.
+Utilizing this folder doesn't count against the 50&nbsp;GB set aside for your plan:
+it's infrastructure disk, including the `/tmp` directory.
+It's included in the metrics so you can keep track of build artifacts you write to `/tmp`
+and make sure that they don't get too large and fill up.
+
+Next to this space comes the other defined directories: for the MySQL service and the `exports` mount.
+
+![A closeup of the metrics for the MySQL service disk](/images/metrics/disk-single.png "0.4")
+
+### Grid environment example
+
+Metrics for Grid environments show the resource usage per container within the project.
+
+This reference project has configured a single application and two services: Redis and MariaDB.
+The plan size for this project is [X-Large](https://platform.sh/pricing/),
+which has guaranteed resources of 8.22 vCPU/12.2&nbsp;GB RAM.
+The appropriate resources have been [allocated automatically](../configuration/app/app-reference.md#sizes) for each container
+based on the number and type of containers for this plan size.
+The graphs show the current average usage in relation to the allocated resources.
+
+#### App container
+
+The app container has been granted 0.75&nbsp;GB of memory and 5.00 CPU.
```

Comment:
Replace with

### Application container

The application metrics graphs show CPU, RAM and Disk allocation and usage.
The persistent disk has been configured in the [app configuration](../configuration/app/app-reference.md#top-level-properties)
at 144.12&nbsp;GB, while the temporary disk is 3.99&nbsp;GB by default.

We do not want to focus on allocation as per https://platformsh.slack.com/archives/CPJNGJ84R/p1644510320745699